src/config.js

```diff
@@ -0,0 +1,3 @@
+export const CAPTCHA_KEY = '6LfPZGAUAAAAANY4ZLR_lSat_YCuqqcV3sl84ulW';
```

Comment:
Only `MAP_KEY` is required. Please remove the other two keys. Thanks :-)

Reply:
@Akshat-Jain Removed It